src/Autofac.Multitenant/MultitenantContainer.cs

```diff
@@ -514,6 +514,22 @@ public ILifetimeScope GetTenantScope(object tenantId)
         }
 
         /// <summary>
+        /// Returns collection of all registered tenant IDs.
+        /// </summary>
+        public ICollection<object> GetTenantsIds()
```

Comment:
We should return an `IEnumerable<object>` to enforce the notion that you can't modify this collection and have the set of configured tenants change.

I think it should probably also be `GetTenants` since everything else is like `RemoveTenant` or `ClearTenants`. It'd be more consistent.